init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
+
+set INIT_TOOLS_LOG=%~dp0init-tools.log
+if [%PACKAGES_DIR%]==[] set PACKAGES_DIR=%~dp0packages\
```

Comment:
The CLI's nuget package is set to `.nuget`.  This should use the same.  We only need 1 package cache.